README.md

```diff
@@ -70,6 +70,10 @@ to the `dependencies` in your `component.json`.
   - **macOS:** no extra step required
 - [Bundler](http://bundler.io/) (Run `gem install bundler` to install).
 - [Node Package Manager (aka. `npm`)](https://docs.npmjs.com/getting-started/installing-node)
+- Tools required to build the font:
+  - **Ubuntu:** `sudo apt-get install fontforge woff-tools woff2`
+  - **Windows:** TODO
+  - **macOS:** TODO
```

Comment:
Either add proper text (instead of TODO) or remove those 2 lines please.

Reply:
Just removed these two lines.

Reply:
thanks.